examples/scale/frontend/frontend.go

```diff
@@ -67,31 +70,58 @@ func Run() {
 }
 
 func doCreate(cfg config.View, fe pb.FrontendClient) {
-	concurrent := cfg.GetInt("testConfig.concurrentCreates")
-	start := time.Now()
+	cycleCreate := func(wg *sync.WaitGroup, ticketPerRoutine int, start time.Time) {
+		defer wg.Done()
+		var cycleCreated uint32
+
+		for j := 0; j < ticketPerRoutine; j++ {
+			req := &pb.CreateTicketRequest{
+				Ticket: tickets.Ticket(),
+			}
+
+			ctx, span := trace.StartSpan(context.Background(), "scale.frontend/CreateTicket")
+			defer span.End()
+
+			timeLeft := start.Add(time.Second).Sub(time.Now())
+			if timeLeft <= 0 {
+				break
+			}
+			ticketsLeft := uint32(ticketPerRoutine) - cycleCreated
+
+			time.Sleep(timeLeft / time.Duration(ticketsLeft))
+
+			if _, err := fe.CreateTicket(ctx, req); err != nil {
+				errMsg := fmt.Sprintf("failed to create a ticket: %w", err)
+				errRead, ok := errMap.Load(errMsg)
+				if !ok {
+					errRead = 0
+				}
+				errMap.Store(errMsg, errRead.(int)+1)
+			}
+			cycleCreated += 1
+		}
+
+		atomic.AddUint32(&totalCreated, cycleCreated)
+	}
+
 	for {
+		if !activeScenario.ShouldCreateTicketForever && int(totalCreated) >= activeScenario.CreateTicketNumber {
```

Comment:
CreateTicketNumber is only used when ShouldCreateTicketForever is false.  This means the domain of possible values for these two variables does not map cleanly to the state space of options.

Also, CreateTicketNumber doesn't have immediate meaning to me.  Perhaps "TotalTicketsToCreate".

Two options I see:
- If TotalTicketsToCreate is zero, that signifies to create infinite tickets.  This reduces it down to a single simple type, but it's not a natural mapping (zero is a lot less than infinity)
- A function such as "ShouldContinueCreatingTickets(totalCreated)" might work.  Though it makes the control surface less obvious, now I need to implement a `func(int) bool` in my scenario, though it could have useful implementations such as "createInfiniteTickets" or "createLimitedTickets"?

Though want is the point of having this variable?  What is the scenario for when it is set?